docs/src/pages/demos/app-bar/BottomAppBar.js

```diff
@@ -0,0 +1,141 @@
+import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import Avatar from '@material-ui/core/Avatar';
+import MenuIcon from '@material-ui/icons/Menu';
+import AddIcon from '@material-ui/icons/Add';
+import SearchIcon from '@material-ui/icons/Search';
+import MoreIcon from '@material-ui/icons/MoreVert';
+
+const styles = theme => ({
+  container: {
+    position: 'relative',
+    maxWidth: 500,
+  },
+  text: {
+    paddingTop: theme.spacing.unit,
+    paddingLeft: theme.spacing.unit * 2,
+    paddingRight: theme.spacing.unit * 2,
+  },
+  main: {
+    height: 300,
+    overflowY: 'auto',
+    position: 'relative',
+  },
+  list: {
+    marginBottom: theme.spacing.unit * 2,
+  },
+  subHeader: {
+    backgroundColor: '#fff',
+  },
+  toolbar: {
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  fabButton: {
+    position: 'absolute',
+    top: -30,
+    left: 0,
+    right: 0,
+    margin: '0 auto',
+  },
+});
+
+const messages = [
+  {
+    id: 1,
+    primary: 'Brunch this week?',
+    secondary: "I'll be in the neighbourhood this week. Let's grab a bite to eat",
+  },
+  {
+    id: 2,
+    primary: 'Birthday Gift',
+    secondary: `Do you have a suggestion for a good present for John on his work 
+      anniversary. I am really confused & would love your thoughts on it.`,
+  },
+  {
+    id: 3,
+    primary: 'Recipe to try',
+    secondary: 'I am try out this new BaqBQ recipe, I think this might be amazing',
+  },
+  {
+    id: 4,
+    primary: 'Yes!',
+    secondary: 'I have the tickets to the ReactConf for this year.',
+  },
+  {
+    id: 5,
+    primary: 'Doctors Appointment',
+    secondary: 'My appointment for the dentist was rescheduled for next Saturday.',
+  },
+  {
+    id: 6,
+    primary: 'Discussion',
+    secondary: `Menus that are generated by the bottom app bar (such as a bottom 
+      navigation drawer or overflow menu) open as bottom sheets at a higher elevation 
+      than the bar.`,
+  },
+  {
+    id: 7,
+    primary: 'Summer BBQ',
+    secondary: `Who wants to have a cookout this weekend? I just got some furniture
+      for my backyard and would love to fire up the grill.`,
+  },
+];
+
+function BottomAppBar(props) {
+  const { classes } = props;
+
+  return (
+    <div className={classes.container}>
+      <Paper className={classes.main}>
+        <Typography className={classes.text} variant="h5" gutterBottom>
+          Inbox
+        </Typography>
+        <List className={classes.list}>
+          <ListSubheader className={classes.subHeader}>Today</ListSubheader>
+          {messages.map(({ id, primary, secondary }) => (
+            <ListItem key={id} button>
+              <Avatar alt="Remy Sharp" src="/static/images/remy.jpg" />
```

Comment:
Another nice-to-have, but perhaps we could have a couple more avatar images, and add them to the messages object? Does the recipient only have one friend? 😆

Reply:
There are only 2 images of people `remy.jpg` and `uxceo-128.jpg` in the `images/` folder, Can I add more image assets in their, maybe the profile pictures of the core members of `Material-UI`?